src/components/cms/SkillCreator/Animation.min.css

```diff
@@ -1 +1,5 @@
 .iconbutton,.iconbutton>div,.iconbutton>div>div,.iconbutton>svg{transition:none!important}
+.__react_component_tooltip.type-dark {
```

Comment:
Why do you do it again and again? When we are moving away from CSS, why use it?

When we already have Tooltip component in Material-UI, then why go for fancy and poor implementations.